pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -794,6 +795,10 @@ func (adc *attachDetachController) VolumeAttachmentLister() storagelistersv1.Vol
 // code to be implemented by all initializers even if the initializer does not
 // do mounting (like this attach/detach controller).
 // Issue kubernetes/kubernetes/issues/14217 to fix this.
+func (adc *attachDetachController) GetTracerProvider() trace.TracerProvider {
+	return nil
```

Comment:
Why is there no TracerProvider here?

Reply:
The `attachdetach` package is used by `kube-controller-manager`, and since this pr is focusing on add trace for csi-plugin in `kubelet`, so I do not implement the `GetTracerProvider` interface used by `kube-controller-manager.`

Reply:
Please use a Noop instead of nil

Reply:
Done